package.json

```diff
@@ -19,7 +19,10 @@
     "test": "TZ=UTC mocha --reporter dot 'lib/**/*.test.js'",
     "test-check-coverage": "npm run test-coverage && nyc check-coverage --branches 100 --functions 100 --lines 100",
     "test-coverage": "nyc --all --reporter text --reporter html --reporter lcovonly npm run test",
-    "demo": "mocha demo/*.test.js"
+    "demo": "mocha demo/*.test.js",
+    "preversion": "npm test-check-coverage",
+    "version": "changes",
```

Comment:
No `--commits` and no `--footer`?

Reply:
I used `node_modules/.bin/changes --init` to generate the entries ...